tests/test_model/test_deeponet.py

```diff
@@ -21,16 +21,19 @@ def test_constructor():
              aggregator='*')
 
 
-def test_constructor_fails_when_invalid_inner_layer_size():
-    branch_net = FeedForward(input_dimensions=1, output_dimensions=10)
-    trunk_net = FeedForward(input_dimensions=2, output_dimensions=8)
-    with pytest.raises(ValueError):
-        DeepONet(branch_net=branch_net,
-                 trunk_net=trunk_net,
-                 input_indeces_branch_net=['a'],
-                 input_indeces_trunk_net=['b', 'c'],
-                 reduction='+',
-                 aggregator='*')
+# This test is wrong! The user could define a custom network and do a 
+# reshape at the end! A more general way to check input and output is
+# needed
+# def test_constructor_fails_when_invalid_inner_layer_size():
+#     branch_net = FeedForward(input_dimensions=1, output_dimensions=10)
+#     trunk_net = FeedForward(input_dimensions=2, output_dimensions=8)
+#     with pytest.raises(ValueError):
+#         DeepONet(branch_net=branch_net,
+#                  trunk_net=trunk_net,
+#                  input_indeces_branch_net=['a'],
+#                  input_indeces_trunk_net=['b', 'c'],
+#                  reduction='+',
+#                  aggregator='*')
```

Comment:
I do not understand why the check is removed. What does it mean `The user could define a custom network and do a reshape at the end`?